core/conn/odb/src/odb.c

```diff
@@ -9889,6 +9889,7 @@ static void Oextract(int eid)
             if ( etab[eid].fho ) {
                 (*hdfswrite)(hfs, etab[eid].fho, (void *)xbuff, xbuffl);
             } else {
+                etab[eid].fo = stdout;
```

Comment:
If the file handler is null, then we may don't need to force to write the message to output. If it has to(the message is really important), a better and professional way to ensure it happen is to initialize the fields of etab when it is allocated. I noticed lots of the fields are pointers, to initialize them requires a lots of work, but it is the safe and professional way.

etab is allocated in etabnew() at line 5974, after the call of realloc and tested no NULL, the next thing should be initialize them all, especially the pointer fields.

@xwq You are more familiar with ODB than I am, any comments?